daal4py/sklearn/linear_model/_coordinate_descent.py

```diff
@@ -736,7 +732,7 @@ def dual_gap_(self, value):
     def dual_gap_(self):
         self._gap = None
 
-    score = support_usm_ndarray()(ElasticNet_original.score)
+    score = ElasticNet_original.score
```

Comment:
Python class inheritance should work fine without this line
```suggestion
```

Reply:
Indeed python class inheritance should work for this. Good catch. These changes are the result of blind removal of the decorator. In reality, there is no need for this assignments. Thank you